src/media-groups.js

```diff
@@ -522,12 +526,17 @@ export const initialize = {
 
         let properties = mediaGroups[type][groupId][variantLabel];
 
+        // TODO: we should filter for hls as well.
```

Comment:
We should probably do the same of HLS playlists, but we don't have a good way to do that right now.

Reply:
Looks like HLS does the same. The codec param for the subs playlist will indicate `stpp.ttml.im1t` From https://tools.ietf.org/html/draft-pantos-hls-rfc8216bis-07.html#section-4.4.6.2

      CODECS

      ...

      Note that if a Variant Stream specifies one or more Renditions
      that include IMSC subtitles, the CODECS attribute MUST indicate
      this with a format identifier such as "stpp.ttml.im1t".

      Every EXT-X-STREAM-INF tag SHOULD include a CODECS attribute.